pkg/addon/addon.go

```diff
@@ -196,6 +196,10 @@ func GetPatternFromItem(it Item, r AsyncReader, rootPath string) string {
 		if strings.HasPrefix(relativePath, strings.Join([]string{rootPath, p.Value}, "/")) {
 			return p.Value
 		}
+		if strings.HasPrefix(relativePath, filepath.Join(rootPath, p.Value)) {
+			// for enable addon by load dir, compatible with linux or windows os
+			return p.Value
+		}
```

Comment:
are they duplicated?

Reply:
no, we use this func for all type source(git,oss, local), if type is git oss linux local will meet first if-case, windows will meet second.

Reply:
I see